tests/test_cli/test_add/test_connection.py

```diff
@@ -511,3 +511,107 @@ def test_add_connection_latest_version(self):
         items_folders = os.listdir(items_path)
         item_name = "local"
         assert item_name in items_folders
+
+
+class TestAddConnectionMixedWhenNoLocalRegistryExists:
+    """Test that the command 'aea add connection' works in mixed mode when the local registry does not exists (it swaps to remote)."""
```

Comment:
Here or below, add the reason how the test works (i.e. in the temporary directory, we don't have the local registry `packages/`, neither in the AEA project nor in the parent directory)